button.html

```diff
@@ -7,7 +7,9 @@
 </head>
 
 <body>
```

Comment:
Why script tag is present inside body tag ?

Reply:
changed the  script tag location to the head tag.